korman/exporter/material.py

```diff
@@ -304,6 +304,18 @@ def export_texture_slot(self, bo, bm, hsgmat, slot, idx, name=None, blend_flags=
             elif slot.blend_type == "MULTIPLY":
                 state.blendFlags |= hsGMatState.kBlendMult
 
+        # Check if this layer uses diffuse/runtime lighting
+        if not slot.use_map_color_diffuse:
+            layer.preshade = hsColorRGBA(0.0, 0.0, 0.0, 1.0)
+            layer.runtime = hsColorRGBA(0.0, 0.0, 0.0, 1.0)
+
+        # Check if this layer uses specular lighting
+        if slot.use_map_color_spec:
+            state.shadeFlags |= hsGMatState.kShadeSpecular
+        else:
+            layer.specular = hsColorRGBA(0.0, 0.0, 0.0, 1.0)
```

Comment:
Magic number?

Reply:
Do we have premade hsColorRGBA values (e.g. "black")?

Reply:
There are already several spots throughout this file that use "magic colours" (mostly white and black).

In this case, if we're not using specular lighting, the specular colour should be set to black to match PlasmaMax.

Reply:
Whoops, I misread this as saying specular is always black